.github/workflows/publish.yml

```diff
@@ -0,0 +1,68 @@
+name: Wheel building
+
+on:
+  workflow_dispatch:
+  push:
+  pull_request:
+
+jobs:
+
+  test_wheel_building:
+    # This ensures that a couple of target wheels build fine in pull
+    # requests and pushes
+    uses: OpenAstronomy/github-actions-workflows/.github/workflows/publish.yml@v1
+    if: (github.event_name == 'push' || github.event_name == 'pull_request')
+    with:
+      upload_to_pypi: false
+      test_extras: test
+      test_command: pytest -p no:warnings --pyargs photutils
+      targets: |
+        - cp310-manylinux_x86_64
+        - cp310*macosx_x86_64
+
+  build_and_publish:
+    # This job builds the wheels and publishes them to PyPI for all
+    # tags, except those ending in ".dev". For PRs with the "Build all
+    # wheels" label, wheels are built, but are not uploaded to PyPI.
+    uses: OpenAstronomy/github-actions-workflows/.github/workflows/publish.yml@v1
+
+    if: (github.repository == 'astropy/photutils' && (github.event_name == 'workflow_dispatch' || contains(github.event.pull_request.labels.*.name, 'Build all wheels')))
+    with:
+      test_extras: test
+      test_command: pytest -p no:warnings --pyargs photutils
+      targets: |
+        # Linux wheels
+        - cp38-manylinux_i686
+        - cp39-manylinux_i686
+        - cp310-manylinux_i686
+        - cp38-manylinux_x86_64
+        - cp39-manylinux_x86_64
+        - cp310-manylinux_x86_64
+        - cp38-musllinux_x86_64
+        - cp39-musllinux_x86_64
+        - cp310-musllinux_x86_64
```

Comment:
Tom R been having issues with musllinux on astropy but hopefully you have better luck here. I just don't even bother with it for synphot...

Reply:
OK, I'll remove it if it has issues.